### PR TITLE
Restructure documentation for bootstrappers

### PR DESCRIPTION
**Pull request by pascalberger**
Restructures documentation for bootstrappers.

- Add documentation how to obtain example bootstrapper for .NET Framework runner, previously available in the [Setting Up A New Project](https://cakebuild.net/docs/tutorials/setting-up-a-new-project) page.
- Merge tutorial about possibility to extend bootstrapper, but get rid of concrete example, since it is quite specific and also out of date compared with current bootstrapping scripts.
- Integrate tutorial about PowerShell security. 

Part of #878

### CODE REVIEW

**Comment by gep13** [comment]
@pascalberger are you in a position to rebase this PR?  Thanks

**Comment by pascalberger** [comment]
@gep13 Rebased and ready for review

File: input/docs/getting-started/bootstrapping-scripts.md [hunk 1]
```diff
@@ -50,3 +50,108 @@ dotnet cake
...
 You can override this behavior by additionally passing the name of the build script.
 :::
+
+# Bootstrapping for Cake runner for .NET Framework
+
+## Getting the bootstrapper
+
+Bootstrapper for [Cake runner for .NET Framework](running-cake-scripts/#cake-runner-for-net-framework) is available in the
+[Cake Resources repository](https://github.com/cake-build/resources) and can be installed using the command for your
+oparting system from below:
```
Comment (gep13) [inline input/docs/getting-started/bootstrapping-scripts.md:60]:
```suggestion
operting system from below:
```

**Comment by gep13** [comment]
@pascalberger your changes have been merged, thanks for your contribution 👍

**Pull request merged**